API/api/services/slot_service.py

```
@@ -0,0 +1,60 @@
+                start_at=data["start_at"],
+                end_at=data["end_at"], 
+                client_fname=data["client_fname"], 
+                client_lname=data["client_lname"], 
+                client_email=data["client_email"], 
+                client_phone=data["client_phone"], 
+                item=data["item"], 
+                item_description=data["item_description"], 
+                liability_accepted=data["liability_accepted"])
+            )
+        except IntegrityError:
+            raise ValidationError({"non_field_errors": "Cette plage horaire ne peut pas être réservée."})
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)